taxonomies/categories.txt

```diff
@@ -28295,6 +29317,64 @@ fi:Suklaa-aamiaismurot
 fr:Céréales au chocolat
 nl:Chocolade ontbijtgranen
 
+<en:Chocolate cereals
+en:Breakfast chocolate cereals with wheat grain flakes (not fortified with vitamins and chemical elements)
```

Comment:
we don't support parenthesis in the taxonomy , this will make "not fortified with vitamins and chemical elements" a synonym of "Breakfast chocolate cereals with wheat grain flakes" --> we should rename the ( ) and just concatenate the string:

Breakfast chocolate cereals with wheat grain flakes not fortified with vitamins and chemical elements

Reply:
OK, I will hunt down all parentheses I left in the text file then. Thanks!